hibernate-spatial/src/test/java/org/hibernate/spatial/testing/dialects/postgis/PostgisTestSupport.java

```diff
@@ -36,7 +36,7 @@ public NativeSQLTemplates templates() {
 	}
 
 	@Override
-	public PredicateRegexes predicateRegexes(){ return new PostgisPredicateRegexes();}
+	public PredicateRegexes predicateRegexes(){ return new PredicateRegexes("st_geomfromtext");}
```

Comment:
Shouldn't it be this?

```suggestion
	public PredicateRegexes predicateRegexes(){ return new PredicateRegexes("st_geomfromewkt");}
```